zerver/tests/test_auth_backends.py

```diff
@@ -474,7 +477,7 @@ class SocialAuthBase(ZulipTestCase):
     def setUp(self) -> None:
         self.user_profile = self.example_user('hamlet')
         self.email = self.user_profile.email
-        self.name = 'Hamlet'
+        self.name = self.user_profile.full_name
```

Comment:
This `self.name` refactor could probably be extracted as a prep commit separate from the main SAML one.

Reply:
extracted to another commit.